src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RequestTracingConstants.cs

```diff
@@ -36,6 +36,8 @@ internal class RequestTracingConstants
         public const string FeatureFlagUsesVariantConfigurationReferenceTag = "ConfigRef";
         public const string FeatureFlagFeaturesKey = "FFFeatures";
         public const string SignalRUsedTag = "UsesSignalR";
+        public const string LoadBalancingEnabledTag = "UsesLoadBalancing";
```

Comment:
I think we need to adopt a pattern like we did here:

https://github.com/Azure/AppConfiguration-DotnetProvider/pull/540#discussion_r1673021342

And do something like `Features=LB`.

Reply:
Also signal r could be reduced. Features=LB+SignalR

Reply:
This pattern `Features=LB` looks good to me.

The signalR telemetry has been released in 7.3.0. Also signalR is not a feature from App Config. I prefer to not include SignalR in `Features`.

Reply:
Do you think it is necessary to move "UsesKeyVault" and "RefreshKeyVault" here?

Reply:
Discussed offline. Some of the data that we put into correlation context we already use in our Power BI charts. This pattern will not include existing fields.